src/Query/QueryBuilder.php

```diff
@@ -146,7 +146,7 @@ public function addFilter(Filter $filter): QueryBuilder
     public function setFilters(array $filters): QueryBuilder
     {
         $this->filterCollection = array_map(function (Filter $filter) {
```

Comment:
`$this->filterCollection = $filters` is equivalent and more performant.

Do you use the `array_map` to type-hint the array content ?

Reply:
I used it because that's how it was, but your comment is true